spec/Section 1 -- Overview.md

```diff
@@ -38,30 +38,30 @@ GraphQL has a number of design principles:
 
  * **Hierarchical**: Most product development today involves the creation and
    manipulation of view hierarchies. To achieve congruence with the structure
-   of these applications, a GraphQL operation itself is structured hierarchically.
-   The operation is shaped just like the data it returns. It is a natural
-   way for clients to describe data requirements.
+   of these applications, a GraphQL request itself is structured
+   hierarchically. The request is shaped just like the data it returns. It is
+   a natural way for clients to describe data requirements.
```

Comment:
> I imagine you're trying to avoid introducing the concept of operation too early

That's exactly right - and here we're looking for a common term as we introduce GraphQL's principles as we compare it to other API systems

Reply:
Sure; resolving 👍